src/EDF.jl

```diff
@@ -132,7 +132,7 @@ struct FileHeader
 end
 
 """
-    EDF.SignalHeader
+    EDF.Signal
```

Comment:
I actually prefer `SignalHeader`; why the change back?

Reply:
My thinking was that this would make transitioning from the previous tagged versions a little smoother by keeping the same struct name, but `SignalHeader` is definitely more descriptive, and it may be to our benefit in this case to have the breaking changes be loud so nothing fails silently; I can totally switch it back.

Reply:
Yeah, I'd say keep it. It's fine to break things if we bump the version number correctly. Anyway, I think `SignalHeader` makes even MORE sense given this particular change.

Reply:
Fixed in https://github.com/beacon-biosignals/EDF.jl/pull/26/commits/6163c31f3a4cb42399643f92820edb8950b670d7